src/CSI.IBTA.Administrator/wwwroot/js/logo-display.js

```diff
@@ -1,10 +1,16 @@
 ﻿function getEncodedLogo(encodedLogo) {
-    document.getElementById('logo-employer').src = "data:image/png;base64," + encodedLogo;
+    if (encodedLogo != null && encodedLogo != "") {
+        try {
```

Comment:
instead of try catch, you could have just used onerror attribute on image element which does the same, for example:

`<img id="imagePreview" src="data:image/png;base64,@Model.EncodedLogo" onerror="this.style.display='none';">`